docs/how_to_guides/create_and_edit_expectations/how_to_create_expectations_between_tables.rst

```diff
@@ -0,0 +1,38 @@
+.. _tutorial_create_expectations_between_tables:
+
+Create Expectations Between Tables
```

Comment:
"How to create Expectations between tables"

* Start with "How to..."
* Use sentence case, not title case

Reply:
Updated